.github/workflows/linux_builds.yml

```diff
@@ -59,6 +59,16 @@ jobs:
             # Skip 2GiB artifact speeding up action.
             artifact: false
 
+          - name: Editor with ThreadSanitizer (target=editor, tests=yes, dev_build=yes, use_tsan=yes, use_llvm=yes, linker=lld)
+            cache-name: linux-editor-thread-sanitizer
+            target: editor
+            tests: true
+            sconsflags: dev_build=yes use_tsan=yes use_llvm=yes linker=lld
+            bin: "./bin/godot.linuxbsd.editor.dev.x86_64.llvm.san"
```

Comment:
The binary name is the same as `linux-editor-llvm-sanitizers`, is this going to be a problem?

Reply:
That's fine, the binary name is only used to run the tests. What matters is that they have a different cache key (which they do).